lib/scrabble/player.rb

```diff
@@ -0,0 +1,50 @@
+module Scrabble
+  class Player
+    attr_accessor :name, :played_words, :playing
+
+    def initialize(name)
+      @name = name
+      @played_words = []
+      @playing = true
+    end
+
+    def play(word)
+      if @playing
+        @played_words.push(word)
+      else
+        return false
+      end
+    end
+
+    def plays
+      return @played_words
+    end
+
+    def total_score
+      final_score = 0
+
+      @played_words.each do |word|
+        final_score += Scrabble.score(word)
+      end
+
+      return final_score
```

Comment:
Good use of an accumulator! For extra brownie points, check out the `#reduce` method in Enumerable (also called `#inject`).